lib/install/management/store_files.go

```diff
@@ -51,18 +51,17 @@ func (d *Dispatcher) DeleteStores(vchVM *vm.VirtualMachine, conf *config.Virtual
 	}
 
 	var errs []string
-	var emptyImages bool
 	var emptyVolumes bool
 	log.Infof("Removing images")
-	if emptyImages, err = d.deleteImages(ds, p); err != nil {
-		errs = append(errs, err.Error())
+	if err = d.deleteImages(conf); err != nil {
+		return err
 	}
 	emptyVolumes, err = d.deleteDatastoreFiles(ds, path.Join(p, volumeRoot), d.force)
 
-	if emptyImages && emptyVolumes {
+	if emptyVolumes {
 		// if not empty, don't try to delete parent directory here
 		log.Debugf("Removing stores directory")
-		if err = d.deleteParent(ds, p); err != nil {
+		if _, err = d.deleteParent(ds, p); err != nil {
```

Comment:
Remove this entire function and replace it with just the call to `deleteImages`. This
a. is trying to delete directories that don't exist
b. shouldn't be deleting the volumes even if it was looking in the correct location.